airflow/www/views.py

```diff
@@ -1271,7 +1271,7 @@ def task_stats(self, session: Session = NEW_SESSION):
 
         if conf.getboolean("webserver", "SHOW_RECENT_STATS_FOR_COMPLETED_RUNS", fallback=True):
             last_dag_run = (
-                select(DagRun.dag_id, sqla.func.max(DagRun.logical_date).label("logical_date"))
+                select(DagRun.dag_id, sqla.func.max(DagRun.run_after).label("run_after"))
```

Comment:
“Last” is defined by logical_date so we should not change this (and some other changes below)

Reply:
ok, I'll revert these UI changes

Reply:
I just reverted these UI changes. but we might need to create an issue for tracking asset triggered dag runs (or any dag runs with null logical date) will not be shown on UI